frontend/src/common/PageWithTabs.tsx

```diff
@@ -116,19 +121,33 @@ export default function PageWithTabs({
         direction={{ xs: 'column', sm: 'row' }}
         divider={<Divider flexItem orientation='vertical' />}
         alignItems='center'
-        spacing={{ sm: 2 }}
+        spacing={{ xs: 1, sm: 2 }}
         sx={{ px: 2, pb: 2 }}
       >
-        <Stack direction='row'>
-          <Typography component='h1' color='primary' variant='h6'>
-            {title}
-          </Typography>
-          {showCopyButton && (
-            <CopyToClipboardButton
-              textToCopy={textToCopy ? textToCopy : title}
-              notificationText='Copied to clipboard'
-              ariaLabel='copy to clipboard'
-            />
+        <Stack>
+          <Stack direction='row'>
+            <Typography component='h1' color='primary' variant='h6'>
+              {title}
+            </Typography>
+            {showCopyButton && (
+              <CopyToClipboardButton
+                textToCopy={titleToCopy ? titleToCopy : title}
+                notificationText='Copied to clipboard'
+                ariaLabel='copy to clipboard'
+              />
+            )}
+          </Stack>
+          {subheading && (
+            <Stack direction='row' alignItems='center'>
+              <Typography variant='caption' sx={{ color: darken(theme.palette.primary.main, 0.4) }}>
+                {subheading}
+              </Typography>
+              <CopyToClipboardButton
+                textToCopy={subheadingToCopy ? subheadingToCopy : subheading}
+                notificationText='Copied to clipboard'
+                ariaLabel='copy to clipboard'
+              />
```

Comment:
Does this component need to be conditional on `showCopyButton`, like with the title page?

Reply:
Yes it does, good spot!